### PR TITLE
Implement editableTitle trl. This fixes duplicate title-tags

### PR DESCRIPTION
**Pull request by Ben-Ho**

### CODE REVIEW

File: Kwc/Box/TitleEditable/Trl/Events.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?php
+class Kwc_Box_TitleEditable_Trl_Events extends Kwc_Chained_Trl_Events
```
Comment (Ben-Ho) [inline Kwc/Box/TitleEditable/Trl/Events.php]: There is no Events.php in Kwc_Box_Title_Trl_Component. Should there be one?

File: Kwc/Box/TitleEditable/Trl/Component.php [hunk 2]
```diff
@@ -1,15 +1,56 @@
...
+        $ret['ownModel'] = 'Kwf_Component_FieldModel';
         return $ret;
     }
 
     public function getIncludeCode()
     {
         return $this->getData();
     }
+
+    protected function _getTitle()
```
Comment (Ben-Ho) [inline Kwc/Box/TitleEditable/Trl/Component.php]: This function is a collection from Kwc_Box_TitleEditable_Component::_getTitle and Kwc_Box_Title_Component::_getTitle(). Maybe this should be solved differently...

**Comment by fraxachun** [comment]
I created another pull request which solves the problem simplier: https://github.com/koala-framework/koala-framework/pull/549

File: Kwc/Box/TitleEditable/Trl/Component.php [hunk 3]
```diff
@@ -1,15 +1,56 @@
...
+        return $ret;
+    }
+
+    public function injectIntoRenderedHtml($html)
+    {
+        return self::injectTitle($html, $this->getData()->render());
+    }
+
+    //public for trl
+    public static function injectTitle($html, $title)
```
Comment (nsams) [inline Kwc/Box/TitleEditable/Trl/Component.php]: don't copy this

File: Kwc/Box/TitleEditable/Trl/Form.php [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<?php
+class Kwc_Box_TitleEditable_Trl_Form extends Kwc_Box_TitleEditable_Form
```
Comment (nsams) [inline Kwc/Box/TitleEditable/Trl/Form.php]: own form, show original text

File: Kwc/Box/TitleEditable/Trl/Events.php (hunk 1 above)
Comment (nsams) [inline Kwc/Box/TitleEditable/Trl/Events.php]: yes

**Comment by nsams** [comment]
@Ben-Ho what's the status for this?

**Comment by Ben-Ho** [comment]
@nsams i think retest and if still defect implement what you commented and update.

but as it was not important enough to be solved since end of july maybe it should just be closed...

**Pull request closed without merging**